cypress/support/step_definitions/authorised/CheckoutProductSteps.ts

```diff
@@ -0,0 +1,74 @@
+import { When } from "@badeball/cypress-cucumber-preprocessor";
+
+const values = {
+  register: [
+    "Becca",
+    "Maxaradze",
+    `${Math.random().toString(6)}@gmail.com`,
+    "example company",
+    "Test12345",
+    "Test12345",
+  ],
+  checkout: ["New York", "Brodway", "NCR", "3423244", "99945323454"],
+};
+
+function registerNewUser() {
+  cy.visit("/register");
+
+  //select gender
+  cy.get(`.male input`).check();
+
+  cy.get(
+    ".form-fields .inputs input:not([type='radio'],[type='checkbox'])"
+  ).each((input, index) => {
+    cy.wrap(input).type(values.register[index]);
+  });
+
+  //select date
+  cy.get("select[name='DateOfBirthDay']").select("8");
+  cy.get("select[name='DateOfBirthMonth']").select("September");
+  cy.get("select[name='DateOfBirthYear']").select("2002");
+
+  cy.get("#register-button").click();
+  cy.wait(2000);
```

Comment:
Please try not to hardcode waits, you can implement cypress-wait-until plugin if necessary